routes/index.js

```diff
@@ -1,171 +1,204 @@
+                            title: 'Family Example',
+                            challenge: pkce_pair['code_challenge']
+                        });
+                    });
+                } else {
+                    res.render('index', {family: family, user: req.session.user, title: 'Family Example', challenge: pkce_pair['code_challenge']});
+                }
+            }).catch((err) => {
+            console.log("in error");
+            console.error(JSON.stringify(err));
+        });
+    } else {
+        res.render('index', {family: family, user: req.session.user, title: 'Family Example', challenge: pkce_pair['code_challenge']});
+    }
 });
 
 /* OAuth return from FusionAuth */
 router.get('/oauth-redirect', function (req, res, next) {
-  // This code stores the user in a server-side session
-  client.exchangeOAuthCodeForAccessToken(req.query.code,
-                                         clientId,
-                                         clientSecret,
-                                         'http://localhost:3000/oauth-redirect')
-      .then((response) => {
-        req.session.state = req.query.state;
-        return client.retrieveUserUsingJWT(response.response.access_token);
-      })
-      .then((response) => {
-        req.session.user = response.response.user;
-      })
-      .then((response) => {
-        if (req.session.state == "confirm-child-list") {
-          res.redirect(302, '/confirm-child-list');
-          return
-        }
-        res.redirect(302, '/');
-     
-      }).catch((err) => {console.log("in error"); console.error(JSON.stringify(err));});
-      
+    // This code stores the user in a server-side session
+    client.exchangeOAuthCodeForAccessTokenUsingPKCE(req.query.code,
+        clientId,
+        clientSecret,
+        'http://localhost:3000/oauth-redirect',
+        req.session.verifier)
+        .then((response) => {
+            req.session.state = req.query.state;
+            return client.retrieveUserUsingJWT(response.response.access_token);
+        })
+        .then((response) => {
+            req.session.user = response.response.user;
+        })
+        .then((response) => {
+            if (req.session.state == "confirm-child-list") {
+                res.redirect(302, '/confirm-child-list');
+                return
+            }
+            res.redirect(302, '/');
+
+        }).catch((err) => {
+        console.log("in error");
+        console.error(JSON.stringify(err));
+    });
+
 });
 
 /* Confirm child list flow */
 router.get('/confirm-child-list', function (req, res, next) {
-  if (!req.session.user) {
-    // force signin
-    res.redirect(302, '/');
-  }
-  client.retrievePendingChildren(req.session.user.email)
-      .then((response) => {
-        res.render('confirmchildren', {children: response.response.users, title: 'Confirm Your Children'});
-      }).catch((err) => {console.log("in error"); console.error(JSON.stringify(err));});
+    if (!req.session.user) {
```

Comment:
Is this just extra spacing? It doesn't look like there is any non whitespace changes. (Or indenting for new blocks.) 

If so, please don't change it.

Reply:
It's just indenting for new blocks. Done by the IDE.